src/utils/prepareSwapTransactions.ts

```diff
@@ -39,6 +39,7 @@ export async function prepareSwapTransactions({
   networkId,
   walletAddress,
   simulatedGasPadding,
+  enableSwapFee,
```

Comment:
the aave swap-short shortcut needs to pass this from the request to this function, same for allbridge (although it doesn't work)